src/utils/helpers/getDateDifference.js

```diff
@@ -0,0 +1,7 @@
+export const getDateDifference = (from) => {
+  let date = new Date(from);
+  let to = new Date();
+  let Difference_In_Time = to.getTime() - date.getTime();
+  let Difference_In_Days = Difference_In_Time / (1000 * 3600 * 24);
```

Comment:
IMHO, Could we use camelCase for variables ?
```suggestion
  let differenceInTime = to.getTime() - date.getTime();
  let differenceInDays = differenceInTime / (1000 * 3600 * 24);
```